clar2wasm/tests/wasm-generation/comparison.rs

```diff
@@ -0,0 +1,33 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use clarity::vm::Value;
+use proptest::proptest;
+
+use crate::string_ascii;
+
+fn filter_invalid_char(input: &Value) -> Value {
+    let invalid_chars: [char; 1] = ['\''];
+    let v: String = input.clone().expect_ascii().replace(invalid_chars, "");
```

Comment:
Why the array instead of a char?

Reply:
If need a filter for any other invalid char the array is already in place.